### ISSUE TITLE
Can't serialize/unserialize more than once

### ISSUE THREAD

**Issue by df999** [opis/closure#40]
Hey there, for some reason, it returns boolean true when I try to serialize an object the 2nd time in a row.

Example:
```php
$x = (object)['foo'=>'bar'];

$s = Opis\Closure\serialize($x);
echo $s; // O:8:"stdClass":1:{s:3:"foo";s:3:"bar";}   (CORRECT)

$u = Opis\Closure\unserialize($s);
$s2 = Opis\Closure\serialize($u);
echo $s2; // b:1;   (WRONG - why isn't it the same as above?)
```

Why does the second serialization produce boolean true?

I'm using the latest opis/closure with PHP 7.3.8 on Win10.

Thanks!

**Comment by msarca** [comment]
This is pretty weird. When I've tried to run that code in a PHPUnit test, the error was there. But when I ran the exactly same code outside PHPUnit, it worked. Are you running this code as part of a PHPUnit test?

**Comment by df999** [comment]
Thanks for looking into it!

Not in unit tests, no. I'm actually just running it in an old Codeigniter setup. I think it was working for the longest time, but then something happened and I can't imagine what I changed (other than regular customer development). I did change the data structure I was serializing a bit, but since this fails on the simplest of objects, I don't think that could be it.

I didn't change any Opis Closure related code or includes so it must be something else...
